scout/adapter/mongo/query.py

```diff
@@ -623,41 +619,41 @@ def secondary_query(self, query, mongo_query, secondary_filter=None):
                             {"length": {"$exists": False}},
                         ]
                     }
-                    LOG.debug("Adding size less than, undef inclusive to query.")
 
                 mongo_secondary_query.append(size_query)
 
             if criterion == "svtype":
                 svtype = query["svtype"]
                 mongo_secondary_query.append({"sub_category": {"$in": svtype}})
-                LOG.debug("Adding SV_type %s to query" % ", ".join(svtype))
 
             if criterion == "decipher":
                 mongo_query["decipher"] = {"$exists": True}
-                LOG.debug("Adding decipher to query")
 
             if criterion == "depth":
-                LOG.debug("add depth filter")
                 mongo_secondary_query.append({"tumor.read_depth": {"$gt": query.get("depth")}})
 
             if criterion == "alt_count":
-                LOG.debug("add min alt count filter")
                 mongo_secondary_query.append({"tumor.alt_depth": {"$gt": query.get("alt_count")}})
 
             if criterion == "control_frequency":
-                LOG.debug("add minimum control frequency filter")
                 mongo_secondary_query.append(
                     {"normal.alt_freq": {"$lt": float(query.get("control_frequency"))}}
                 )
 
             if criterion == "tumor_frequency":
-                LOG.debug("add minimum VAF filter")
                 mongo_secondary_query.append(
                     {"tumor.alt_freq": {"$gt": float(query.get("tumor_frequency"))}}
                 )
 
             if criterion == "mvl_tag":
-                LOG.debug("add managed variant list filter")
                 mongo_secondary_query.append({"mvl_tag": {"$exists": True}})
```

Comment:
Do you think this might also need a `$ne: None` soon, or is that just being extra careful for cosmic etc?

Reply:
I had to add it for cosmic otherwise the query didn't work: many vars had cosmic_ids:none. I'm not sure how mvl_tag looks like

Reply:
It should be fine, unless there is a hit. Not sure if one would be able to get to an existing but None key with adding and removing MVs. Or by a slightly mislead pydantic application..

Reply:
I think it's fine: mvl_tag exists only if there is a hit, for instance the demo cases have no variants with mvl_tag.

Reply:
It will be evident then, dont worry.

Reply:
Just my stream of consciousness speaking - and checking if I had missed something.